CPDShell/CPDShell.py

```diff
@@ -0,0 +1,86 @@
+from typing import Optional
+
+from .Core.CPDCore import *
+from .Core.scrubber.scrubber import Scrubber
+
+
+class MarkedCPData:
+    """Class for generating and storing marked data,
+    needed in CPDShell
+
+    :param raw_data: Data for CPD algorithms
+    :type raw_data: list[float]
```

Comment:
Types in docstring are redundant (we use type annotations) and require maintenance